Advanced topics (#154)
RMI-PACTA/pacta.loanbook: vignettes/cookbook_advanced_use_cases.Rmd

      TRUE ~ sector
    )
  )

regions <- region_isos_demo
scenario <- scenario_demo_2020

market_share_targets_portfolio <-
  target_market_share(
    data = matched,
    abcd = abcd,
    scenario = scenario,
    region_isos = regions
  )

market_share_targets_company <-
  target_market_share(
    data = matched,
    abcd = abcd,
    scenario = scenario,
    region_isos = regions,
    by_company = TRUE,
    weight_production = FALSE
  )

co2 <- co2_intensity_scenario_demo

sda_targets_portfolio <-
  target_sda(
    data = matched,
    abcd = abcd,
    co2_intensity_scenario = co2,
    region_isos = regions
  )

sda_targets_company <-
  target_sda(
    data = matched,
    abcd = abcd,
    co2_intensity_scenario = co2,
    region_isos = regions,
    by_company = TRUE
  )

```

# Potential Follow-up Actions based on PACTA for Banks analysis

PACTA for Banks analyses can be useful across a diverse set of real-life applications. In this section, we aim to highlight some of the use cases that we consider particularly suitable for the type of analysis at hand. All these use cases are currently either being used or explored by PACTA users.

> The types of use cases we explore in this section all build on the standard PACTA for Banks outputs – that is they use results based on the market share approach or the sectoral decarbonization approach – either at loan book level or at company level. They vary by the intended target audience – generally either an internal audience (another department in the FI or management) or an external audience (often either a client or a regulator/supervisor, sometimes the broader public). Additionally, the use cases vary by type, where some are focused more on communicating/reporting and others are more focused on implementing change or informing internal decisions.

Author: Do I understand correctly that there is no immediate action to be taken in this PR though?

> The types of use cases we explore in this section all build on the standard PACTA for Banks outputs – that is they use results based on the market share approach or the sectoral decarbonization approach – either at loan book level or at company level. They vary by the intended target audience – generally either an internal audience (another department in the FI or management) or an external audience (often either a client or a regulator/supervisor, sometimes the broader public). Additionally, the use cases vary by type, where some are focused more on communicating/reporting and others are more focused on implementing change or informing internal decisions.

Reviewer: Yeah let's say no action required, and we can make a new issue for this.

We will explain in some detail some of the most established use cases in this cookbook, but we will mention others only briefly, especially ones that have not been explored in quite as much detail yet and that would therefore require additional research before describing potential ways to implement them.

## Use Case 1: Regulatory compliance

### Description

Report or disclose loan book alignment or targets to a financial supervisor or regulator

### Level of analysis

Regulators will normally not mandate reporting on individual clients and normally leave the choice of clients up to the financial institution. Instead, rules and regulations apply to the structure of the overall loan book. Hence, in most cases, calculating PACTA for Banks results at the loan book level will be the appropriate depth of analysis for this use case.

### Potential application

**Overview**

> Financial regulators across a variety of jurisdictions in the world have introduced regulations to account for climate-related financial risks in recent years. The exact specifications of regulatory requirements differ on a case-by-case basis. But in many cases, the regulations involve reporting on the materiality of climate-relevant exposures and in some cases they include reporting on forward-looking metrics, such as alignment of emissions or production to an external benchmark or emissions/production targets of the financial institution to ensure climate transition trends are accounted for by financial institutions. In all cases, requirements can vary by type of climate metric to report on (e.g. absolute emissions, financed emissions, production-based emission intensity, production volumes), sectoral scope and granularity (financial scope and granularity: all exposures combined at the loan book level, aggregate exposures to clients in climate-relevant sectors only, sector-level exposures to climate relevant sectors; sectoral scope: reporting on all parts of the value chain or on the ones most relevant for emissions).

Reviewer: > Financial regulators across a variety of jurisdictions in the world have introduced regulations to account for climate-related financial risks in recent years

Would be good to give at least once example here, if we have one? I wonder if referencing the EBA makes sense here?

Author: Valid point, I do think it needs careful wording though, as I think we don't want to say that we are sure that PACTA will be sufficient when reporting to the EBA. Maybe good to sync with George on this topic

Reviewer: Hmmm ok. Let's not let that block this PR then, and we can consider that a potential addition if we are confident we can do so 😊

```{r use_case_one_materiality_calculation}
materiality <- coverage %>% 
  mutate(sector = if_else(!sector %in% c("power", "cement"), "not in scope", sector)) %>% # we assume this bank only lends to cement, power, and not-in-scope companies
  summarise(
    sector_sum_loan_size_outstanding = sum(loan_size_outstanding),
    .by = sector
  ) %>%
  mutate(
    sector_share_loan_size_outstanding = round(sector_sum_loan_size_outstanding / sum(sector_sum_loan_size_outstanding), 2)
  )
```

```{r use_case_one_materiality_result, echo = FALSE}
result_table(materiality, title = "Materiality assessment", subtitle = "loan book exposures to climate relevant sectors")
```

> The bank might report to the supervisor along the lines of:
> -	`r pull(filter(materiality, sector == "power"), sector_share_loan_size_outstanding) * 100`% of the loan book (`r pull(filter(materiality, sector == "power"), sector_sum_loan_size_outstanding)` USD) go towards power generation

Reviewer 2: add a newline here to make the following show as bullet points

```suggestion
The bank might report to the supervisor along the lines of:

-	`r pull(filter(materiality, sector == "power"), sector_share_loan_size_outstanding) * 100`% of the loan book (`r pull(filter(materiality, sector == "power"), sector_sum_loan_size_outstanding)` USD) go towards power generation
```

> -	`r pull(filter(materiality, sector == "power"), sector_share_loan_size_outstanding) * 100`% of the loan book (`r pull(filter(materiality, sector == "power"), sector_sum_loan_size_outstanding)` USD) go towards power generation

Reviewer 2: `as.character()` should make it print as a rounded integer

```suggestion
-	`r pull(filter(materiality, sector == "power"), sector_share_loan_size_outstanding) * 100`% of the loan book (`r as.character(pull(filter(materiality, sector == "power"), sector_sum_loan_size_outstanding))` USD) go towards power generation
```

> -	`r pull(filter(materiality, sector == "cement"), sector_share_loan_size_outstanding) * 100`% of the loan book (`r pull(filter(materiality, sector == "cement"), sector_sum_loan_size_outstanding)` USD) go towards cement production
> -	`r pull(filter(materiality, sector == "not in scope"), sector_share_loan_size_outstanding) * 100`% of the loan book (`r pull(filter(materiality, sector == "not in scope"), sector_sum_loan_size_outstanding)` USD) go towards other sectors that are not climate relevant

Reviewer 2: `as.character()` should make it print as a rounded integer

```suggestion
-	`r pull(filter(materiality, sector == "cement"), sector_share_loan_size_outstanding) * 100`% of the loan book (`r as.character(pull(filter(materiality, sector == "cement"), sector_sum_loan_size_outstanding))` USD) go towards cement production
-	`r pull(filter(materiality, sector == "not in scope"), sector_share_loan_size_outstanding) * 100`% of the loan book (`r as.character(pull(filter(materiality, sector == "not in scope"), sector_sum_loan_size_outstanding))` USD) go towards other sectors that are not climate relevant
```

> This can be a direct outcome based on the matched and prioritized data set compared with values in the raw loan book data set. (((Note that if sectors in adjacent parts of the value chain are required too, it may be an idea to include loans with both flags: borderline TRUE and FALSE.)))

Reviewer 2: is this supposed to be wrapped in `(((` `)))`?
```suggestion
This can be a direct outcome based on the matched and prioritized data set compared with values in the raw loan book data set. (Note that if sectors in adjacent parts of the value chain are required too, it may be an idea to include loans with both flags: borderline TRUE and FALSE.)
```

Author: tbh, I am a bit uncertain about this section/comment. It feels like explaining this properly would take significantly more space. I put it in ((())) to note to myself that this should either be explained more or removed.

Author: moved this whole paragraph to "Notes and caveats" section

    year == 2030
  ) %>% 
  select(sector, emission_factor_metric, year, emission_factor_value) %>% 
  mutate(emission_factor_value = round(emission_factor_value, 3))

ei_cement <- ei_cement_current %>% 
  bind_rows(ei_cement_target) %>% 
  bind_rows(ei_cement_scenario)
```

The outcome would look as follows:

```{r use_case_one_alignment_targets_result, echo = FALSE, warning = FALSE}
result_table(tech_mix_power, title = "Power sector alignment and targets", subtitle = "high-carbon share")

result_table(ei_cement, title = "Cement sector alignment and targets", subtitle = "emission intensity")
```

And accordingly, the bank might report:

- The current high-carbon share (coal-fired, gas-fired and oil-fired power plants) of the companies the bank lends to in the power sector is: `r sum(filter(tech_mix_power_current, technology %in% c("coalcap", "gascap", "oilcap"))$technology_share, na.rm = TRUE) * 100`%
- The target high-carbon share in the power sector that the bank has set itself based in the "target_sda" scenario for 2030 is: `r sum(filter(tech_mix_power_target, technology %in% c("coalcap", "gascap", "oilcap"))$technology_share, na.rm = TRUE) * 100`%

> - The current emission intensity of the companies the bank lends to in the cement sector is: `r pull(ei_cement_current, emission_factor_value)` tCO2/t cement
> - The target emission intensity in the cement sector that the bank has set itself based in the "target_demo" scenario for 2030 is: `r pull(ei_cement_target, emission_factor_value)` tCO2/t cement
> - The scenario value of the emission intensity in the cement sector in the "adjusted_scenario_demo" scenario for 2030 is: `r pull(ei_cement_scenario, emission_factor_value)` tCO2/t cement

Reviewer 2: maybe better to spell these out?

```suggestion
- The current emission intensity of the companies the bank lends to in the cement sector is: `r pull(ei_cement_current, emission_factor_value)` tonnes of CO^2^/tonne of cement
- The target emission intensity in the cement sector that the bank has set itself based in the "target_demo" scenario for 2030 is: `r pull(ei_cement_target, emission_factor_value)` tonnes of CO^2^/tonne of cement
- The scenario value of the emission intensity in the cement sector in the "adjusted_scenario_demo" scenario for 2030 is: `r pull(ei_cement_scenario, emission_factor_value)` tonnes of CO^2^/tonne of cement
```

- The target high-carbon share in the power sector is: `r sum(filter(tech_mix_sector_targets, sector == "power", technology %in% c("coalcap", "gascap", "oilcap"))$technology_share, na.rm = TRUE) * 100`%
- If targets are set by technology in the power sector, the targets would look as follows:
  - share of coal-fired power plants: `r sum(filter(tech_mix_sector_targets, sector == "power", technology == "coalcap")$technology_share, na.rm = TRUE) * 100`%
  - share of gas-fired power: `r sum(filter(tech_mix_sector_targets, sector == "power", technology == "gascap")$technology_share, na.rm = TRUE) * 100`%
  - share of oil-fired power: `r sum(filter(tech_mix_sector_targets, sector == "power", technology == "oilcap")$technology_share, na.rm = TRUE) * 100`%
  - share of renewable power: `r sum(filter(tech_mix_sector_targets, sector == "power", technology == "renewablescap")$technology_share, na.rm = TRUE) * 100`%

>   - share of hydro power: `r sum(filter(tech_mix_sector_targets, sector == "power", technology == "hydrocap")$technology_share, na.rm = TRUE) * 100`%

Reviewer: More user-friendly to display this as a table

Author: @jdhoffa I made a new issue for this, we can beautify in a follow up PR #161

> - The target emission intensity in the cement sector is: `r pull(filter(ei_sector_targets, sector == "cement"), emission_factor_value)` tCO2/t cement
> - The target emission intensity in the steel sector is: `r pull(filter(ei_sector_targets, sector == "steel"), emission_factor_value)` tCO2/t steel

Reviewer 2: maybe better to spell these out?

```suggestion
- The target emission intensity in the cement sector is: `r pull(filter(ei_sector_targets, sector == "cement"), emission_factor_value)` tonnes of CO^2^/tonne of cement
- The target emission intensity in the steel sector is: `r pull(filter(ei_sector_targets, sector == "steel"), emission_factor_value)` tonnes of CO^2^/tonne of steel
```
